tests/rename_test.go

```diff
@@ -2,6 +2,11 @@ package tests
 
 import (
 	"testing"
+	"time"
+
+	"github.com/emersion/go-imap"
```

Comment:
More go style advice: imports should be grouped in two clusters separated by a newline, the first cluster should be stdlib stuff, the second cluster should be community stuff. Seems you've added these imports with unnecessary newlines around them.